src/app.js

```diff
@@ -28,3 +28,5 @@ app.use("/", routes);
 app.listen(process.env.PORT, () => {
     console.info(`Server listening on port ${process.env.PORT}`);
 });
+
+module.exports = app;
```

Comment:
Why are you exporting this?
Maybe it's a good practice or something I don't know about, but I don't think it will ever be used

Reply:
I think this code was already written so I'm not sure. maybe @miguelpduarte can help us?

Reply:
This was added in this PR so I am confused xd

Reply:
you are right. I must have written it when I was trying to change the file for the faculties endpoint. It is not mandatory so maybe I should remove it?